src/components/views/elements/SSOButtons.tsx

```diff
@@ -62,6 +84,8 @@ const SSOButton: React.FC<ISSOButtonProps> = ({
     const label = idp ? _t("Continue with %(provider)s", { provider: idp.name }) : _t("Sign in with single sign-on");
 
     const onClick = () => {
+        const authenticationType = getAuthenticationType(idp.brand);
+        PosthogAnalytics.instance.setAuthenticationType(authenticationType);
```

Comment:
Keep in mind SSOButtons are also used for UIA, imo it shouldn't setAuthenticationType when going via UIA

Reply:
After discussing offline, it looks like User-Interactive Auth will render a single SSO button, so that won't be a problem.
And in other places, when people click on an SSO button, it seems to be a strong intent to authenticate with that specific provider